glusterfs/orchestrate/server.sls

```diff
@@ -0,0 +1,18 @@
+glusterfs.server.service:
+  salt.state:
+    - tgt: 'glusterfs:server'
+    - tgt_type: pillar
+    - queue: True
+    - sls: glusterfs.server.service
+    - batch: 1
```

Comment:
Using batch run when it's not necessary will result in much slower deployment/updates.
And in this case it's really not needed.